StaticSiteGenerators(tables).rst

```diff
@@ -14,42 +14,40 @@ Starting notes:
 
-Advantages:
------------
+Advantages of SSDs:
+-------------------
 
 * You can leverage developer tools like Github.
-* Markup is straightforward to read and learn. easier to use than HTML. rST, Markdown, and AsciiDoc are examples of markup files. 
-* Easy migrate from one static site generator to another. 
-* There are lots of free, open source tools.
-* Uses a workflow that developer are familiar with.  
+* You author content for SSDs in markup languages, which are straightforward to read and learn, and easier to use than raw HTML. rST, Markdown, and AsciiDoc are examples of markup languages.
+* It's relatively easy to migrate from one static site generator to another.
+* There are lots of free, open source SSD tools.
+* SSDs use a workflow that developers are familiar with.
 
-One important advantage is that you can keep the documentation really close to the code and you can use the same workflow as for code. This encourages developers to keep documentation synchronized with the code. 
+One important advantage is that you can keep the documentation close to the code, sometimes even in the same repository as the code, and you can use the same workflow as for code. This encourages developers to keep documentation synchronized with the code.
 
 You should use a static site generator if:
 ------------------------------------------
 
-* You need to collaborate with developers. 
-* Relatively easy to set up.
-* Can be free.
+* You need to collaborate with developers.
+* You need something that's easy to set up and to maintain.
+* Your budget for infrastructure and tooling is low.
 
 Disadvantages:
 --------------
 
-* No content management system to manage:
-* Permissions - Access control isn’t built into most static site generators. If you don’t want your docs publicly accessible, access control is an additional expense on your time. 
-* Versioning - If you’re using a certain kind of markup, you can use readthedocs has versioning built into the theme. Different branches represent different versions. If you name the branch a certain way (for example 3.5). Whenever you’re ready to release a different version of your product, you create a different branch, it creates the version for you.
-* No one has developed this function for Markdown yet. Or it’s not popular yet. 
-* How do you go from version to version and how easy is it for your end users to find different versions of your docs. 
+* SSDs have no content management system to manage things like:
+  * Permissions - Access control isn’t built into most static site generators. If you don’t want your docs publicly accessible, access control is an additional expense on your time.
+  * Versioning - Not all SSDs support deploying different versions of the same content at once. However, some SSDs can use version control to handle deploying different versions simultaneously.
```

Comment:
There's more information in the versioning list item in the original text that I'm not sure bears removing. What are your thoughts on adding another list item or additional explanation outside of the list?

Reply:
Ok, I've added back some of that info on versioning. Not sure where else it should go other than that same bullet.